client/src/main/java/jp/co/soramitsu/iroha/java/QueryBuilder.java

```diff
@@ -21,22 +25,62 @@
 import iroha.protocol.Queries.QueryPayloadMeta;
 import iroha.protocol.Queries.TxPaginationMeta;
 import java.time.Instant;
+import java.util.ArrayList;
 import java.util.Date;
 import java.util.List;
 import java.util.stream.Collectors;
+import jp.co.soramitsu.iroha.java.crypto.Ed25519Sha3SignatureBuilder;
+import jp.co.soramitsu.iroha.java.crypto.SignatureBuilder;
+import lombok.Getter;
 import lombok.val;
 
 public class QueryBuilder {
 
+  public class Ordering {
+
+    class Sequence {
```

Comment:
it seems lombok's `@Value` could be used instead of explicit constructor and `@Getter` declarations

Reply:
ok